PoShLucene.ps1

```diff
@@ -208,10 +350,7 @@ function DoSearch ($q)
 			$totalHits = $isearcher.Search($query, $null, 1000).ScoreDocs
 		}
 
-
-		$totalHits
-		$txtStatus.text = "{0}. {1} hits found in {2} seconds" -f ($txtStatus.text -split "\. ")[0], $totalHits.count, $timing.TotalSeconds
```

Comment:
So I noticed that $txtStatus.text was being set twice here and removed one instance.  Was that a mistake?